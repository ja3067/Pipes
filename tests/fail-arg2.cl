def dynamic():
	if True:
		return 3
	else:
		return 4.0

def foo(x : float):
	return x

print(foo(dynamic()))

#def foo():
#	x : int  = dynamic()
#	return x
#
#print(foo())
