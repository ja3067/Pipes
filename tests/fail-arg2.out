RuntimeError: invalid type assigned to x
